architecture/overview.md

```diff
@@ -0,0 +1,66 @@
+---
+title: Architecture: Overview
+summary: 
+toc: false
+---
+
+## Goals
+
+CockroachDB was designed in service of the following goals:
+
+- Make life easier for humans. This means being low-touch and highly automated for operators and simple to reason about for developers.
+- Provide scale without sacrificing consistency. This means enabling distributed transactions, as well as removing the pain of eventual consistency issues and stale reads.
+- Create an always-on database that continues accepting reads and writes without creating any conflicts.
+- Allow flexible deployment on any cloud, without tying you to any platform.
+- Leverage the power of orchestration tools.
+
+## Glossary
+
+### Terms
+
+It's helpful to understand a few terms when reading our architecture documentation.
+
+Term | Definition
+-----|-----------
+**Cluster** | Your CockroachDB deployment, which acts as a single logical database
+**Node** | An individual server running CockroachDB; you join many nodes together to create a cluster
+**Range** | A contiguous set of data from your cluster
+**Replicas** | Copies of your ranges, which are stored on at least 3 nodes to ensure survivability.
+
+### Concepts
+
+CockroachDB heavily relies on the following concepts, so understanding them will help you understand what our architecture achieves.
+
+Term | Definition
+-----|-----------
+**Consistency** | CockroachDB uses this term in a few different ways, but most commonly as consistency in terms of the [CAP theorem](https://en.wikipedia.org/wiki/CAP_theorem). In this context, consistency means that every read receives the most recent write or an error. Most often we refer to consistency as the CAP theorem guarantee we prioritize in face of all others, making us what's known as "CP" database.<br/><br/>However, CockroachDB does also also support [consistency in terms of ACID semantics](https://en.wikipedia.org/wiki/Consistency_(database_systems)). To confuse matters slightly, what we typically refer to as consistency (in the CAP theorem sense), is also tightly coupled to our support of [isolation in terms of ACID semantics](https://en.wikipedia.org/wiki/Isolation_(database_systems)).
+**Consensus** | When an action is performed on a range, a quorum of nodes containing a replica of it acknowledge the action. When an action *doesn't* achieve consensus, forward progress halts to ensure consistency within the cluster.
```

Comment:
I'd say that DDL statements can be described as "writes" too (and they do require quorum), and it would be more clear to say "writes" instead of "actions"

Reply:
Done.